src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -203,8 +203,11 @@ function plant_absorbed_pfd(
 
     (; ld, Ω, n_layers) = RT.parameters
 
+    # Get the current leaf angular distribution value based on the solar zenith angle
+    G = FT(ld(cos(θs)))
```

Comment:
I wonder if we can use a better name than G? or is G common notation for this problem?

Reply:
G-funtion is the name. Ross, J. (1981), The Radiation Regime and Architecture of Plant Stands, Dr. W. Junk, The Hague, Netherlands.